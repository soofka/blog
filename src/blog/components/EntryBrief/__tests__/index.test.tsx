import { shallow } from 'enzyme';
import * as React from 'react';

import { EntryBrief } from '../';

describe('<EntryBrief/>', () => {

  it('renders correctly', () => {
    expect(shallow(
      <EntryBrief
        text="testText"
      />,
    )).toMatchSnapshot();
  });

  it('renders correctly with optional props', () => {
    expect(shallow(
      <EntryBrief
        text="testText"
        imageFileName="testImageFileName"
        moreButtonLink="testMoreButtonLink"
      />,
    )).toMatchSnapshot();
  });

});
