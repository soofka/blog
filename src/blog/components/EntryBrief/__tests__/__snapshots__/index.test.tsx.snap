// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EntryBrief/> renders correctly 1`] = `
<div>
  <styled.div
    dangerouslySetInnerHTML={
      Object {
        "__html": "testText",
      }
    }
  />
  <p />
</div>
`;

exports[`<EntryBrief/> renders correctly with optional props 1`] = `
<div>
  <styled.div>
    <styled.img
      src="/images/testImageFileName"
    />
  </styled.div>
  <styled.div
    dangerouslySetInnerHTML={
      Object {
        "__html": "testText",
      }
    }
  />
  <p>
    <Link
      replace={false}
      to="testMoreButtonLink"
    >
      <inject-_class-with-languageStore
        name="entry__continue_reading"
      />
       Â»
    </Link>
  </p>
</div>
`;
